File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/smart_pointer_not_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'smart_pointer_not'

include Smart_pointer_not

f = Foo.new
b = Bar.new(f)
s = Spam.new(f)
g = Grok.new(f)

begin
  x = b.x
  puts "Error! b.x"
rescue NameError
end

begin
  x = s.x
  puts "Error! s.x"    
rescue NameError
end

begin
  x = g.x
  puts "Error! g.x"
rescue NameError
end

begin
  x = b.getx()
  puts "Error! b.getx()"    
rescue NameError
end

begin
  x = s.getx()
  puts "Error! s.getx()"        
rescue NameError
end

begin
  x = g.getx()
  puts "Error! g.getx()"
rescue NameError
end

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/smart_pointer_not_runme.py
from smart_pointer_not import *

f = Foo()
b = Bar(f)
s = Spam(f)
g = Grok(f)

try:
    x = b.x
    print "Error! b.x"
except:
    pass

try:
    x = s.x
    print "Error! s.x"
except:
    pass

try:
    x = g.x
    print "Error! g.x"
except:
    pass

try:
    x = b.getx()
    print "Error! b.getx()"
except:
    pass

try:
    x = s.getx()
    print "Error! s.getx()"
except:
    pass

try:
    x = g.getx()
    print "Error! g.getx()"
except:
    pass
